pkg/operator/config/config.go

```diff
@@ -12,8 +12,11 @@ type Config struct {
 	// IngressControllerImage is the ingress controller image to manage.
 	IngressControllerImage string
 
-	// CanaryImage is the ingress canary image to manage.
+	// CanaryImage is the ingress operator image, which runs a canary command.
 	CanaryImage string
 
+	// CanaryCommand is the command to serve a healthcheck via the ingress operator.
+	CanaryCommand string
```

Comment:
I dont think `CanaryCommand` needs to be a part of this struct if we are hard coding it anyways.